.github/workflows/wagon-tests.yml

```diff
@@ -66,6 +66,56 @@ jobs:
         run: |
           bundle exec rake app:rubocop
 
+  wagon_seeds:
+    name: seeds
+    runs-on: 'ubuntu-20.04'
+    env:
+      HEADLESS: true
+      RAILS_DB_ADAPTER: postgresql
+      RAILS_DB_HOST: 127.0.0.1
+      RAILS_DB_PORT: 5432
+      RAILS_DB_USERNAME: hitobito
+      RAILS_DB_PASSWORD: hitobito
+      RAILS_DB_NAME: hitobito_development
+      RAILS_ENV: development
+    # only run seeds on master
+    if: ${{ github.ref_name == 'run-seeds-in-ci' }}
```

Comment:
I experimented with various incantation of this condition to check if I can get it to execute when the https://github.com/hitobito/hitobito_sac_cas/pull/1047 branch triggers the workflow (thus currently `run-seeds-in-ci` later we'd change it to `master`). But it didn't work :( it just always skips...

Reply:
Ahh
![image](https://github.com/user-attachments/assets/f13b626d-d79e-49c1-9471-7998203a886d)

> For pull requests, the format is <pr_number>/merge.